docs/datasources/flow_datasource.adoc

```diff
@@ -0,0 +1,63 @@
+= Flow Datasource
+
+== General
+
+The Flow Datasource can be used to retrieved application and conversation related metrics derived from flow data stored by OpenNMS.
```

Comment:
_Flow Datasource_?

My english is not very good, but shouldn't it state `The _Flow Datasource_ can be used to retrieve application and conversation related metrics derived from flow data stored by OpenNMS`?